src/components/field-types/MultiSelectFieldComponent.vue

```diff
@@ -140,9 +165,15 @@ export default {
     }
   },
   components: {
-    vSelect,
+    Multiselect,
     FormFieldMessages,
     Description
   }
 }
 </script>
+<style scoped>
+  .multiselect {
+    width: 1%;
```

Comment:
`width: 1%;` feels a bit strange, is there an important reason for this?

Reply:
I see the reason now. Feels like the parent element needs some `flex-wrap: nowrap;` but i don't see a nice selector for that.